packages/react/src/components/Tooltip/next/Tooltip.stories.js

```diff
@@ -68,6 +69,22 @@ export const Duration = () => {
   );
 };
 
+export const Definition = () => {
+  const definition =
+    'Starfleet was the deep space exploratory and defense service maintained by the United Federation of Planets.';
+  return (
+    <div>
+      <p>
+        The first duty of every{' '}
+        <DefinitionTooltip definition={definition}>Starfleet</DefinitionTooltip>{' '}
+        officer is to the truth, whether it’s scientific truth, or historical
+        truth, or personal truth! It is the guiding principle on which Starfleet
+        is based.
+      </p>
+    </div>
+  );
+};
```

Comment:
We should try and use example copy that's consistent across docs 👀

Reply:
Star Wars guy, huh?

Reply:
Grabbed the copy from Modal -- lemme know if there's something more apropos